composer.json

```diff
@@ -12,19 +12,26 @@
     ],
     "require": {
         "php": "^7.3",
-        "illuminate/http": "^7.0|^8.0",
-        "illuminate/console": "^7.0|^8.0"
+        "laravel/framework": "^7.0||^8.0"
```

Comment:
I swapped in this requirement for the following reasons:

* `runningInConsole()` is a method only available through `\Illuminate\Contracts\Foundation\Application` while not being available on the illuminate console application
* Orchestra Testbench uses this requirement which conflicts with `illuminate/http` and `illuminate/console`

Reply:
i tried to add as less packages as possible... but this is ok for now.